src/tests/ftest/util/dmg_utils.py

```diff
@@ -855,6 +855,18 @@ def system_query(self, ranks=None, verbose=True):
         self.log.info("system_query data: %s", str(data))
         return data
 
+    def system_leader_query(self):
+        """Query system to obtain the MS leader and replica information.
+
+        Raises:
+            CommandFailure: if the dmg system query command fails.
+
+        Returns:
+            dictionary of output in JSON format
+
+        """
+        return self._get_json_result(("system", "leader-query"))
```

Comment:
It would be better if you could write the system leader-query command output here as a comment, so that others would know what to expect without actually running the command.

Reply:
Good idea. I'll add it.